docs/rules/prefer-arrow-callback.md

```diff
@@ -1,15 +1,22 @@
-# Suggest using arrow functions as callbacks. (prefer-arrow-callback)
+# Suggest using ES6 arrow functions to describe callbacks (prefer-arrow-callback)
 
-Arrow functions are suited to callbacks, because:
+Compared to function expressions, arrow functions are better-suited for describing callbacks or function arguments.
```

Comment:
Could you re-word this sentence somehow? It feels too forceful and opinionated the way it's currently written. We try very hard to avoid forcing our opinions on users.